### PR TITLE
Switch to MIC backend

### PR DESCRIPTION
**Pull request by twicki**
## Technical Description

The naive c++ backend has the loop order 
```
for(stencils, mss)
  for(stages)
    for(k-levels)
      parfor(ij)
```
since our optimization-strategy was based on the GPU backend, where the order is
```
for(stencils, mss)
  for(k-levels)
    for(stages)
      parfor(ij)
```
we had to switch to the MIC backend (which follows the same logic) to produce useful data.

## Content
- Switch to MIC backend to fix numerics
- Beautifying Timer output
- Fix to CMake that was found while developing this

### CODE REVIEW

**Comment by twicki** [comment]
launch jenkins

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -92,9 +92,10 @@ option(GTCLANG_ENABLE_GRIDTOOLS "Enable GridTools compilation" OFF)
 
 set(gtclang_dependencies "boost" "dawn")
 
+message("MOD ${CMAKE_MODULE_PATH}")
```
Comment (cosunae) [inline bundle/CMakeLists.txt]: remove?

Comment (twicki) [reply bundle/CMakeLists.txt]: fixed, sorry

**Comment by cosunae** [comment]
launch jenkins

**Pull request merged**